### PR TITLE
Increase timeout for httpclient because of publishing issues reported for 0.7.3

### PR DESCRIPTION
**Pull request by timbru**

### CODE REVIEW

File: Changelog.md [hunk 1]
```diff
@@ -3,6 +3,12 @@
 Please see [here](https://github.com/NLnetLabs/krill/projects?query=is%3Aopen+sort%3Aname-asc)
 for planned releases.
 
+## 0.7.5 'Time's Up'
+
+This release is not intended for public use, please upgrade to the latest version - which is
+Krill 0.9.3 'Thundering Herd' - if you can. We introduced this version as a quick fix to
+deal with http client timeouts for publishing CAs.
```
Comment (ximon18) [inline Changelog.md:10]: s/http/HTTP/

File: Changelog.md [hunk 2]
```diff
@@ -3,6 +3,12 @@
 Please see [here](https://github.com/NLnetLabs/krill/projects?query=is%3Aopen+sort%3Aname-asc)
 for planned releases.
 
+## 0.7.5 'Time's Up'
+
+This release is not intended for public use, please upgrade to the latest version - which is
+Krill 0.9.3 'Thundering Herd' - if you can. We introduced this version as a quick fix to
```
Comment (ximon18) [inline Changelog.md:9]: This will very soon be outdated as 0.9.4 is being prepared.

File: src/constants.rs [hunk 3]
```diff
@@ -49,4 +49,4 @@ pub const ID_CERTIFICATE_VALIDITY_YEARS: i32 = 15;
 
 pub const BGP_RIS_REFRESH_MINUTES: i64 = 60;
 
-pub const HTTTP_CLIENT_TIMEOUT_SECS: u64 = 120;
+pub const HTTTP_CLIENT_TIMEOUT_SECS: u64 = 900;
```
Comment (ximon18) [inline src/constants.rs:52]: This still seems to be 120 on HEAD of `main`, i.e. the issue doesn't just exist in 0.7.4 but also in newer versions. Is it the intention to make this config file configurable as part of #740?

**Review by ximon18 — APPROVED** [review]

**Review by ximon18 — APPROVED** [review]

**Comment by timbru** [comment]
This was a quick fix for 0.7.4 and should not be merged to main. It has been reimplemented for 0.9.4 and uses a config setting. Closing this PR.

**Pull request closed without merging**